app/assets/javascripts/controllers/container_deployment/deploy_provider/deploy-provider-review-progress-controller.js

```diff
@@ -0,0 +1,13 @@
+miqHttpInject(angular.module('miq.containers.providersModule')).controller('containers.deployProviderReviewProgressController',
+  ['$rootScope', '$scope',
+  function($rootScope, $scope) {
+    'use strict';
+
+    var firstShow = true;
```

Comment:
`firstShow` in `onShow` function is used all across the wizard controllers, to support moving between steps. we plan to add functionality to this substep , in a different PR.
I removed this function for now, thanks.